src/Instrument/ClassLoading/SourceTransformingLoader.php

```diff
@@ -130,13 +138,21 @@ public static function addTransformer(SourceTransformer $transformer)
     public static function transformCode(StreamMetaData $metadata)
     {
         $result = null;
+
         foreach (self::$transformers as $transformer) {
             $result = $transformer->transform($metadata);
             if ($result === false) {
                 break;
             }
         }
 
+        if ($result) {
```

Comment:
If file must be transformed, and we already transformed it -> that is because we have circular weaving.